env.go

```diff
@@ -77,6 +77,9 @@ type StartOptions struct {
 	UserNs           string
 	Privileged       bool
 	Capabilities     []RunnableCapabilities
+
+	LimitMemoryBytes uint
```

Comment:
Optional: I'm wondering if it would not be easier to make this `string` and just allow to also specify the unit, instead of having user always calculate to bytes.

Reply:
Yea, I thought it's easier to do bytes than trying to figure out if we should do `500M` or `500m` or `500MB`. Instead we can do it simply by just `5 * 1024 * 1024` (:

Reply:
happy to change if you prefer string

Reply:
Honestly it's a minor thing, I don't have strong preference. Let's keep it.